workflow/snakemake_rules/core.smk

```diff
@@ -32,7 +32,7 @@ rule exclude_bad:
     params:
         min_date=config["min_date"],
         min_length=config["min_length"],
-        strain_id=lambda w: config.get("strain_id_field", "accession"),
+        strain_id=lambda w: config.get("strain_id_field"),
```

Comment:
It would be best to add a config schema and do a config validation at the start of the workflow like we do in the [ncov workflow](https://github.com/nextstrain/ncov/blob/ba9953f8a61d239176e960b9b2efb925b5dff84b/Snakefile#L45), but that's definitely out of scope for this PR.

Reply:
Thanks for the tip! Done across all files: c10fde839c21c18ec96dad82f66d9c6c5737540c

Reply:
This seems useful to add to the Snakemake style guide: https://github.com/nextstrain/docs.nextstrain.org/pull/163